src/core/Geocoding.js

```diff
@@ -3,40 +3,43 @@ module.exports = function Geocoding(options) {
   var map = options.map || document.getElementById("map") || L.map('map');
 
   function getPlacenameFromCoordinates(lat, lng, precision, onResponse) {
-      $.ajax({
-        url:"https://maps.googleapis.com/maps/api/geocode/json?latlng="+lat+","+lng + "&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE",
-        success: function(result) {
-          if(result.results[0]) {
-            var country;
-            var fullAddress = result.results[0].formatted_address.split(",");
-            for (i in result.results) {
-              if(result.results[i].types.indexOf("country") != -1) {
-                //If the type of location is a country assign it to thr input box value
-                country = result.results[i].formatted_address;
-              }
+    $.ajax({
+      url:"https://maps.googleapis.com/maps/api/geocode/json?latlng="+lat+","+lng + "&key=AIzaSyAOLUQngEmJv0_zcG1xkGq-CXIPpLQY8iQ",
```

Comment:
So i guess we have to change this to the JS version? 

Here seems to be the docs for that: https://developers.google.com/maps/documentation/javascript/geocoding#ReverseGeocoding

Reply:
Ahhh!! Ok! I will get it switched over.